src/interface.js

```diff
@@ -0,0 +1,50 @@
+$(document).ready(function(){
+
+  var scorecard = new Bowling();
+
+  $('#begin').click(function() {
+    $('.mainCard').fadeTo('slow', 2, function(){});
+    $('#start').hide();
+    updateScores();
+  });
+
+  $('.buttons').click(function() {
+    var bowl = Number($(this).val())
+    try {
+      scorecard.bowl(bowl);
+    } catch(err) {
+      alert(err);
+    };
+    buttonManage();
+    updateScores();
+  });
+
+  var buttonManage = function() {
+    // Hacky fix for buttons disappearing on last round
+    // Buttons disappear due to pinsLeft being 0, however
+    // the frame function can't be updated without breaking 
+    // everything.
+    // The 'fix' allows illegal numbers to be entered however.
```

Comment:
Comments when you have surprising behaviour :+1:
Surprising behaviour :disappointed: